packages/manager/apps/telecom/src/app/telecom/pack/move/eligibility/address/move-eligibility-address.service.js

```diff
@@ -0,0 +1,24 @@
+export default class MoveEligibilityAddressService {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  searchStreetNumber(streetCode) {
+    return this.$http
+      .post('/connectivity/eligibility/search/streetNumbers/details', {
+        streetCode,
+      })
+      .then((response) => response);
```

Comment:
Return the same object in the `.then` callback is useless. But here, you can wrap the `data` object response to ease the call back success management  
```suggestion
      .then(({data}) => data);
```